input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,499 @@
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="StructureDefinition-au-core-location.html">AU Core Location</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-location.html">US Core Location</a></td>
+            <td style="width: 25%;">Location.name</td>
+            <td style="width: 25%;">US Core requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="StructureDefinition-au-core-medication.html">AU Core Medication</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-location.html">US Core Location</a></td>
+            <td style="width: 25%;">Medication.code</td>
+            <td style="width: 25%;">US Core uses an extensible binding to <a href="https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1010.4/expansion">Medication Clinical Drug</a>, which is not in AU medicines terminology.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="StructureDefinition-au-core-medication.html">AU Core Medication</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-location.html">US Core Location</a></td>
+            <td style="width: 25%;">MedicationRequest.code</td>
+            <td style="width: 25%;">US Core uses an extensible binding to <a href="https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1010.4/expansion">Medication Clinical Drug</a>, which is not in AU medicines terminology.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-organization.html">AU Core Organization</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-organization.html">US Core Organization</a></td>
+            <td style="width: 25%;">Organization.active</td>
+            <td style="width: 25%;">US Core requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td rowspan="3" style="width: 25%;"><a href="https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-patient.html">AU Core Patient</a></td>
+            <td rowspan="3" style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-patient.html">US Core Patient</a></td>
+            <td style="width: 25%;">Patient.identifier</td>
+            <td style="width: 25%;">US Core requires all identifiers to be valid (have system and value).</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Gender identity</td>
+            <td style="width: 25%;">US Core defines an US Core Gender Identity extension. AU Core uses the International Gender Identity extension.</td>
```

Comment:
Instead of "defines' should be "requires" - rephrase to be "US Core requires US Core Gender Identity extension"

Reply:
Done

Reply:
Remove all "AU Core uses the International Gender Identity extension."

Reply:
Done